drivers/volume/portworx/portworx.go

```diff
@@ -2676,6 +2685,28 @@ func (d *portworx) ToggleCallHome(n node.Node, enabled bool) error {
 	return nil
 }
 
+func (d *portworx) ValidateStorageCluster() error {
+	pxOps, err := pxOperator.ListStorageClusters(schedops.PXNamespace)
+	if err != nil {
+		return err
+	}
+	if len(pxOps.Items) > 0 {
+		pxVersion, err := d.getPxVersionOnNode(node.GetStorageDriverNodes()[0])
```

Comment:
I mean this can work, but I think we need to get PX version from StorageCluster itself, if its edge-install then we should get it from env var PX_RELEASE_MANIFEST_URL and then compare it to what is actually installed on nodes.